docs/pages/blog/2023-toolpad-beta-announcement.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Toolpad is now in Beta!
```

Comment:
How about this for a flashier and more descriptive title? I don't think we've ever published an "official" public announcement about Toolpad, so this could be it.

```suggestion
title: Introducing Toolpad: MUI's low-code admin builder framework
```